docs/content/dagster-cloud/deployment/agents/running-multiple-agents.mdx

```diff
@@ -9,7 +9,7 @@ platform_type: "cloud"
 
 Each Dagster Cloud full deployment (e.g., `prod`) needs to have at least one agent running. Since the agent is lightweight and can tolerate short downtimes such as during an upgrade, a single agent is adequate for most use cases.
 
-If you want to distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments, you can run multiple agents. For example, one in each cluster, instance, etc.
+If you want to distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments, you can run multiple agents isolated across these environments. For example, one in each cluster, instance, etc.
 
 It's recommended to only use multiple agents of the same type (e.g. multiple Kubernetes agents). Submitting runs to a specific agent isn't currently supported.
```

Comment:
i think we should emphasize here that you can't (yet) control which of the isolated agents your request will go to - a lot of the reason people would turn to this guide would be to do that now that it's not about replication

Reply:
assuming this comment is irrelevant now that we've changed scope of the guide